src/components/Chip/styles.css

```diff
@@ -0,0 +1,48 @@
+.chip-row {
```

Comment:
The general layout for chip should be like this
```scss
.chip {
    display: flex;

    .icon {
        flex-shrink: 0;
    }
    .label {
        flex-grow: 1;
    }
    .action {
        flex-shrink: 0;
    }
}
```
```
-----------------------------------------------
| icon | label                       | action |
-----------------------------------------------
```

@safar @AdityaKhatri thoughts?

Reply:
I agree

Reply:
makes sense!